packages/web/src/generic.test.ts

```diff
@@ -39,3 +39,82 @@ describe('inject', () => {
     });
   });
 });
+
+describe('track custom events', () => {
+  beforeEach(() => {
+    // reset the internal queue before every test
+    window.vaq = [];
+  });
+  describe('queue custom events', () => {
+    it('event name only', () => {
+      inject();
+
+      track('my event');
+
+      expect(window.vaq).toBeDefined();
+
+      if (!window.vaq) throw new Error('window.vaq is not defined');
```

Comment:
I think this is redundant as we already have an expect for that on line 54

Reply:
It probably satisfies typescript though.

Reply:
I think so. I'd argue the `toBeDefined` may be redundant, but at least we'll get a nice test failure that way.